apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -112,42 +146,89 @@ const sendSubmitMeterReadingsPushNotifications = async () => {
             offset: state.offset,
             pageSize: state.pageSize,
         })
-        const meterReadings = await readMeterReadings({ context, meters })
         const organizations = await readOrganizations({ context, meters })
+        const reportingPeriods = await readMeterReportingPeriods({ context, organizations })
+        const meterReadings = await readMeterReadings({ context, meters, reportingPeriods })
+
+        const metersWithoutReadings = []
+        const periodsByProperty = []
+        const periodsByOrganization = []
+        let defaultPeriod = null
+
+        for (let period of reportingPeriods) {
+            if (!period.organization) defaultPeriod = period
+            else if (!period.property) periodsByOrganization.push(period)
+            else periodsByProperty.push(period)
+        }
+
+        for (const meter of meters) {
+
```

Comment:
extra lines?

Reply:
Updated) these were logs for finding a bug on the review stand. I just made a lot of lines to see and remove later